pootle/apps/pootle_store/syncer.py

```diff
@@ -235,10 +243,10 @@ def create_store_file(self, last_revision, user):
         store = self.convert()
         if not os.path.exists(os.path.dirname(self.store_file_path)):
             os.makedirs(os.path.dirname(self.store_file_path))
+        self.store.file = self.relative_file_path
```

Comment:
im not sure how it worked correctly previously - but what i know is that sometimes test where using gnu conventions and sometimes nongnu - and sometimes a mixture in tests.

afaict tho you are right that it needs a relative path for `PootleFileSystemStorage`